frontend/locales/en-US.json

```diff
@@ -429,7 +432,46 @@
       "noActivityFound": "No activities found",
       "noLogsFound": "No logs found"
     },
-    "settings": "Settings"
+    "settings": "Settings",
+    "users": {
+      "avatar": "Avatar",
+      "edit": "Edit user",
+      "lastActivityDate": "Last seen {value} ago",
+      "libraryAccess": "Library access",
+      "libraryAccessNote": "Select the libraries to share with this user. Administrators will be able to edit all folders using the metadata manager.",
+      "new": "Create user",
+      "tabs": {
+        "profile": {
+          "allowRemoteConnections": "Allow remote connections to this server",
+          "deleteUser": "Delete user",
+          "deleteUserConfirm": "Are you sure you want to delete this user?",
+          "name": "Name",
+          "profile": "Profile"
+        },
+        "access": {
+          "access": "Access",
+          "allLibraries": "Enable access to all libraries"
+        },
+        "parentalControl": {
+          "addBlockedTag": "Add blocked tag",
+          "blockUnratedItems": "Block items with no or unregocnized rating information:",
+          "blockTags": "Block items with tags:",
+          "maxAllowedRating": "Maximum allowed parental rating",
+          "maxAllowedRatingSubtitle": "Content with a higher rating will be hidden from this user.",
+          "parentalControl": "Parental Control",
+          "tagName": "Tag name",
+          "unblockTag": "Unblock tag"
+        },
+        "password": {
+          "confirmPassword": "Confirm new password",
+          "currentPassword": "Current password",
+          "newPassword": "New password",
+          "password": "Password",
+          "resetPassword": "Reset password"
+        }
+      },
+      "users": "Users"
+    }
```

Comment:
Can you put all those keys at the root level? We're killing off all the nested subkeys. More [here](https://github.com/jellyfin/jellyfin-vue/pull/1985)

Reply:
Can do. Should I just remove the categories and put all the keys on the root level or is there any syntax I should follow? Putton them all on the root level might cause conflicts if I don't rename some of them.

Reply:
@rdmchr Yeah, exactly that.

Are there any conflicts whose value differs from your original text? If not, just reuse the original key!

Reply:
Yeah, some keys could be reused and I didn't encounter any difficult conflicts.